CHANGELOG.md

```diff
@@ -4,6 +4,68 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.13.0 (2023-11-27)
+
+This release contains performance improvements, an improved hypertable DDL API
+and bug fixes since the 2.12.2 release. We recommend that you upgrade at the next
+available opportunity.
```

Comment:
@mkindahl should we add a note on the API changes on the noteworthy features list below? Is there a page we could link to?